.buildkite/scripts/steps/code_generation/elastic_assistant_codegen.sh

```diff
@@ -6,5 +6,5 @@ source .buildkite/scripts/common/util.sh
 
 echo --- Elastic Assistant OpenAPI Code Generation
 
-(cd x-pack/plugins/elastic_assistant && yarn openapi:generate)
+(cd x-pack/packages/kbn-elastic-assistant-common && yarn openapi:generate)
```

Comment:
@jbudz, [feedback](https://github.com/elastic/kibana/pull/174718#pullrequestreview-1818965413) was to move our schemas from plugin -> common package, so this that update.